README.md

```diff
@@ -4,6 +4,50 @@ A module for FeedHenry RainCatcher that manages users, groups and memberships. I
 - Backend services to handle CRUD operations for user, group and membership.
 - Frontend directives and services providing CRUD clients for user, group and membership.
 
+## Upgrading to 0.2.0 from 0.1.x
+Version 0.2.0 introduces session storage for authenticated users utilizing either MongoDB or Redis as storage engines.
+
+This involves an extra parameter to the initialization of the router for the authentication service which contains the configuration for the session storage.
+
+### How to upgrade
+
+In the MBaaS service that authenticates users (e.g. [raincatcher-demo-auth](https://github.com/feedhenry-raincatcher/raincatcher-demo-auth)), initialise the session store with the configuration shown below.
+
+```javascript
+const userRouter = require('fh-wfm-user/lib/router/mbaas');
+
+userRouter.init(
+  mediator, // fh-wfm-mediator instance
+  expressApp, // express application upon which to mount the router
+  ['password'], // list of fields from Users to exclude from the HTTP responses
+
+  // Session storage configuration, new in 0.2.0
+  {
+    store: 'mongo', // The store to utilize for session storage,
+                    // current available values are ['mongo', 'redis']
+    config: {
+      // the following parameters are forwarded to express-session
+      secret: 'raincatcher',
+      resave: false,
+      saveUninitialized: true,
+      cookie: {
+        secure: true,
+        httpOnly: true,
+        path: '/'
+      }
+      // `url` is used by the 'mongo' store for configuration
+      url: 'mongodb://localhost:27017/raincatcher-demo-auth-session-store'
```

Comment:
That's the application/client code's responsibility. In practice we do that in demo auth here: https://github.com/feedhenry-raincatcher/raincatcher-demo-auth/blob/feature/persistent-storage/lib/sessionInit.js
The `$fh.db` call will grab the url from `FH_MONGODB_CONN_URL`